crypto/err/openssl.txt

```diff
@@ -2490,34 +2490,6 @@ EVP_R_WRAP_MODE_NOT_ALLOWED:170:wrap mode not allowed
 EVP_R_WRONG_FINAL_BLOCK_LENGTH:109:wrong final block length
 EVP_R_XTS_DATA_UNIT_IS_TOO_LARGE:191:xts data unit is too large
 EVP_R_XTS_DUPLICATED_KEYS:192:xts duplicated keys
-KDF_R_BAD_ENCODING:122:bad encoding
```

Comment:
Should these kind-of unrelated changes in openssl.txt be in a separate commit? They still can be part of this PR.

Reply:
This actually appears to be caused by issue #10522 fixed by #10549.

Reply:
I've removed these extraneous changes which were an artefact of "make update" out of this PR.